https://issues.redhat.com/browse/ACM-6119 (#5333)
open-cluster-management/rhacm-docs: networking/submariner/install_subctl.adoc

| link:https://submariner.io/operations/deployment/subctl/#version[`version`] | Displays the version details of the `subctl` binary tool.
|===

> *Note*: Only the commands listed above support `subctl`. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Reviewer: Improve this statement for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=medium-accessibility

> *Note*: Only the commands listed above support `subctl`. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Reviewer 2: ```suggestion
*Note*: Red Hat's build of `subctl` only includes commands relevant to ACM deployments. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```

Same feedback as on https://github.com/stolostron/rhacm-docs/pull/5252.

> *Note*: The Red Hat build of `subctl` includes only the commands that are relevant to {product-title}. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Reviewer 3: ```suggestion
*Note*: The Red Hat build of `subctl` only includes the commands that are relevant to {product-title}. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```

> *Note*: `subctl` can only support commands relevant to {rhacm-short} deployments which are listed above. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Author: {rhacm-short} . Need to locate where the shortened version of product names are in docs.

> *Note*: `subctl` can only support commands relevant to {product-title} deployments which are listed above. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Reviewer: @dfarrell07 made a comment in the [previous PR](https://github.com/stolostron/rhacm-docs/pull/5252#discussion_r1262822375): 

<img width="794" alt="Screenshot 2023-08-04 at 8 21 53 AM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/cb65326f-f712-4149-a843-54e37f618305">

Improve this statement for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=medium-accessibility

Words like above, below, left, right should not be used in customer product doc. 

And check for guidance on Possessives: https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

> *Note*: `subctl` can only support commands relevant to {product-title} deployments which are listed above. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Reviewer: ```suggestion
*Note*: The Red Hat build of `subctl` includes only the commands that are relevant to {product-title}. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```

| link:https://submariner.io/operations/deployment/subctl/#version[`version`] | Displays the version details of the `subctl` binary tool.
|===

*Note*: Only the commands listed above support `subctl`. 

> For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Author: ```suggestion

```

|===

For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

> *Note*: Only the commands listed above support `subctl`. 

Author: ```suggestion
*Note*: Only the the most relevant commands, including those listed above, support `subctl`. 
```

Author: This is a suggestion following @dfarrell07 suggestion from the previous PR.

> *Note*: Only the commands listed above support `subctl`. 

Reviewer: Improve this statement for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=medium-accessibility

> For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].

Reviewer: Looks like this is a duplicate statement. This statement is also on line 46.

Author: Thank you!
